### DISCUSSION TITLE
Can I add a team from an external organization to my organization?

### DISCUSSION THREAD

**Discussion by FedPat-CF** [community/community#133659] [New to GitHub] [answered]
I would like to add an external team to collaborate with my organization.

**Comment by SohanKafle** [comment]
Hi [FedPat-CF](https://github.com/FedPat-CF),

Currently, you can't add a team from an external organization directly to your organization. However, you can invite individual members to collaborate.

<h2>For that you need to follow these steps:</h2>
       1. Go to Your Repository: Navigate to the repository.<br>
       2. Invite Collaborators: Settings > Manage access > Invite a collaborator.<br>
       3. Create a Team: Organization > Teams > New team. Add external members to this team.<br>

This way, you can collaborate effectively with external members.

**Reply by FedPat-CF** [reply, marked as answer]
Thanks, @SohanKafle!

**Reply by ghostinhershell** [reply]
Hi @FedPat-CF 👋🏾,It looks like @SohanKafle's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
